docs/40_smart-contract-guides/80_linking-custom-permission.md

```diff
@@ -2,72 +2,585 @@
+cleos push action scholder why '["name"]' -p bob@active
+```
+
+The output should be similar to:
+
+```console
+executed transaction: 43b6ad4ce7a52d7281ccd2800caa02d5278ee714de36fefe9624bff621378402  104 bytes  129 us
+#      scholder <= scholder::why               {"user":"name"}
+>> why not name
+```
+
+```shell
+cleos push action scholder what '["name"]' -p bob@active
+```
+The output should be similar to:
+
+```console
+executed transaction: 43b6ad4ce7a52d7281ccd2800caa02d5278ee714de36fefe9624bff621378402  104 bytes  129 us
+#      scholder <= scholder::what               {"user":"name"}
+>> hi, what do you want name
+```
+
+```shell
+cleos push action scholder how '["name"]' -p bob@active
+```
+
+The output should be similar to:
+
+```console
+executed transaction: 43b6ad4ce7a52d7281ccd2800caa02d5278ee714de36fefe9624bff621378402  104 bytes  129 us
+#      scholder <= scholder::how               {"user":"name"}
+>> how are you name
+```
+
+We see that the `active` permission can sucessfully call all the actions.
+
+Call the actions using the _customp1_ permission:
+
+```shell
+cleos push action scholder why '["name"]' -p bob@customp1
+```
+
+The output should be similar to:
+
+```console
+Error 3090005: Irrelevant authority included
+Please remove the unnecessary authority from your action!
+Error Details:
+action declares irrelevant authority '{"actor":"bob","permission":"customp1"}'; minimum authority is {"actor":"bob","permission":"active"}
+```
 
 ```shell
-cleos set account permission alice upsert YOUR_PUBLIC_KEY owner -p alice@owner
+cleos push action scholder what '["name"]' -p bob@customp1
 ```
-A few things to note:
 
-1. A new permission called **upsert** was created
-2. The **upsert** permission uses the development public key as the proof of authority
-3. This permission was created on the `alice` account
+The output should be similar to:
 
-You can also specify authorities other than a public key for this permission, for example, a set of other accounts. Check [account permission](https://developers.eos.io/manuals/eos/v2.1/cleos/command-reference/set/set-account) for more details.
-## Step 2. Link Authorization to Your Custom Permission
-Link the authorization to invoke the `upsert` action with the newly created permission:
+```console
+executed transaction: 43b6ad4ce7a52d7281ccd2800caa02d5278ee714de36fefe9624bff621378402  104 bytes  129 us
+#      scholder <= scholder::what               {"user":"name"}
+>> why not name
+```
 
 ```shell
-cleos set action permission alice addressbook upsert upsert
+cleos push action scholder how '["name"]' -p bob@customp1
+```
+The output should be similar to:
+
+```console
+executed transaction: 43b6ad4ce7a52d7281ccd2800caa02d5278ee714de36fefe9624bff621378402  104 bytes  129 us
+#      scholder <= scholder::how               {"user":"name"}
+>> how are you name
+```
+
+We see that the _customp1_ permission can sucessfully call the _what_ and _how_ actions, but fails to call the _why_ action.
```

Comment:
For consistency with previous instances:
\_what\_ and \_how\_ -> \`what\` and \`how\`
\_why\_ -> \`why\`